# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/webcodecs/videoDecoder-codec-specific.https.any.js.ini
# Commit: aea093a94985
# Full Hash: aea093a949856e14212082873768c90e664fb595
# Author: Chun-Min Chang <chun.m.chang@gmail.com>
# Date: 2023-06-30 04:06:43
# Regressor Bug: 1831451
# File Overlap Count: 1
# Description:
#   Bug 1831451 - Implement VideoDecoder::Configure r=padenot
#   
#   This patch implements configure method [1] for WebCodecs' VideoDecoder
#   interface. The configure job is via DecoderAgent's Configure().
#   
# ==============================================================================

diff -r 13f8ed81dca8 -r aea093a94985 testing/web-platform/meta/webcodecs/videoDecoder-codec-specific.https.any.js.ini
--- a/testing/web-platform/meta/webcodecs/videoDecoder-codec-specific.https.any.js.ini	Thu Jun 29 22:33:12 2023 +0000
+++ b/testing/web-platform/meta/webcodecs/videoDecoder-codec-specific.https.any.js.ini	Thu Jun 29 22:33:12 2023 +0000
@@ -3,12 +3,6 @@
   expected:
     if (os == "android") and fission: [ERROR, TIMEOUT]
 
-  [Test invalid configs]
-    expected: FAIL
-
-  [Test configure()]
-    expected: FAIL
-
   [Decode a key frame]
     expected: FAIL
 
@@ -54,12 +48,6 @@
   expected:
     if (os == "android") and fission: [ERROR, TIMEOUT]
 
-  [Test invalid configs]
-    expected: FAIL
-
-  [Test configure()]
-    expected: FAIL
-
   [Decode a key frame]
     expected: FAIL
 
@@ -105,12 +93,6 @@
   expected:
     if (os == "android") and fission: [ERROR, TIMEOUT]
 
-  [Test invalid configs]
-    expected: FAIL
-
-  [Test configure()]
-    expected: FAIL
-
   [Decode a key frame]
     expected: FAIL
 
@@ -156,12 +138,6 @@
   expected:
     if (os == "android") and fission: [ERROR, TIMEOUT]
 
-  [Test invalid configs]
-    expected: FAIL
-
-  [Test configure()]
-    expected: FAIL
-
   [Decode a key frame]
     expected: FAIL
 
@@ -207,12 +183,6 @@
   expected:
     if (os == "android") and fission: [ERROR, TIMEOUT]
 
-  [Test invalid configs]
-    expected: FAIL
-
-  [Test configure()]
-    expected: FAIL
-
   [Decode a key frame]
     expected: FAIL
 
@@ -258,12 +228,6 @@
   expected:
     if (os == "android") and fission: [ERROR, TIMEOUT]
 
-  [Test invalid configs]
-    expected: FAIL
-
-  [Test configure()]
-    expected: FAIL
-
   [Decode a key frame]
     expected: FAIL
 
@@ -309,12 +273,6 @@
   expected:
     if (os == "android") and fission: [ERROR, TIMEOUT]
 
-  [Test invalid configs]
-    expected: FAIL
-
-  [Test configure()]
-    expected: FAIL
-
   [Decode a key frame]
     expected: FAIL
 
@@ -360,12 +318,6 @@
   expected:
     if (os == "android") and fission: [ERROR, TIMEOUT]
 
-  [Test invalid configs]
-    expected: FAIL
-
-  [Test configure()]
-    expected: FAIL
-
   [Decode a key frame]
     expected: FAIL
 
@@ -411,12 +363,6 @@
   expected:
     if (os == "android") and fission: [ERROR, TIMEOUT]
 
-  [Test invalid configs]
-    expected: FAIL
-
-  [Test configure()]
-    expected: FAIL
-
   [Decode a key frame]
     expected: FAIL
 
@@ -462,12 +408,6 @@
   expected:
     if (os == "android") and fission: [ERROR, TIMEOUT]
 
-  [Test invalid configs]
-    expected: FAIL
-
-  [Test configure()]
-    expected: FAIL
-
   [Decode a key frame]
     expected: FAIL
 
